docs/man/crystal/README.md

```diff
@@ -379,11 +379,16 @@ Options:
 * `-fFORMAT`, `--format=FORMAT`: Output format `tree` (default), `flat`, `dot`, or `mermaid`.
 * `-iPATH`, `--include=PATH`: Include path in output.
 * `-ePATH`, `--exclude=PATH`: Exclude path in output.
```

Comment:
To be consistent, and also the previous line (GH is not allowing me to pick all the lines). It won't be consistent all over (that's impossible at this point), but at least it will be consistent here.
```suggestion
* `-f FORMAT`, `--format FORMAT`: Output format `tree` (default), `flat`, `dot`, or `mermaid`.
* `-i PATH`, `--include PATH`: Include path in output.
* `-e PATH`, `--exclude PATH`: Exclude path in output.
```